ENIGMAsystem/SHELL/Bridges/Win32-Direct3D11/graphics_bridge.cpp

```diff
@@ -58,7 +58,8 @@ namespace enigma
     enigma_user::draw_clear(enigma_user::window_get_color());
   }
   
-  void EnableDrawing (HGLRC *hRC) {
+  void EnableDrawing(void* handle) {
```

Comment:
Why is `void*` necessary here? Bridges can use window-system–specific types. You can always parameterize your main loop logic, or require an `enigma::window_handle_t`.